templates/Dockerfile-deb.tmpl

```
@@ -115,6 +115,23 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.22.2/cmake-3.22.2
     rm -f cmake-3.22.2.tar.gz && \
     rm -rf cmake-3.22.2
 
+# install python 3.8 to be able to execute tools scripts
+# if os/release in (debian/buster,ubuntu/bionic)
+RUN !( ( [ %%os%% = debian ] && [ %%release%% = buster ] ) || \
+        ( [ %%os%% = ubuntu ] && [ %%release%% = bionic ] ) \
+     ) || ( \
```

```suggestion
     ) && ( \
```

It looks like we install python 3.8 for all the distros other than buster and bionic.The comments indicate that we should do it other way around.

---

Installed pyenv for all distros. Therefore no need to change this. BTW I tested this condition and it works as intended

---

Looks like I missed the negation in `RUN !(`. Thanks for not implementing my suggestion blindly.